packages/react-native-sdk/src/components/call/LobbyView.tsx

```diff
@@ -129,16 +133,22 @@ export const LobbyView = ({ joinCallButton }: LobbyViewType) => {
         )}
         <View style={styles.info}>
           <Text style={styles.infoText}>
-            You are about to join a call with id {call?.id} at Stream.{' '}
-            {participantsCount
-              ? `${participantsCount} participant(s) are in the call.`
-              : 'You are first to Join the call.'}
+            {t('You are about to join a call with id {{ callId }} at Stream.', {
+              callId: call?.id,
+            }) +
```

Comment:
problematic to say at Stream in SDK actually :( 

I actually don't know what to do here expect saying lets pass a prop for info

at stream basically means DF app

Reply:
I think the whole info text can be passed as prop 

Also we can add heading, subheading prop .. 

Expect for Join I don't see any other texts being useful in translations

Reply:
Good find. Let me fix this.

Reply:
About the prop thing, what I believe is probably in the future is to design and keep LobbyView in the app itself and SDK won't export it in my opinion so it's fine for now.